### PR TITLE
Send double-click on appicon to WindowMaker

### PR DESCRIPTION
**Pull request by trunkmaster**
Old code sent to WM only single-click. 
Receiving double-click on GNUstep appicons allows WindowMaker to perform its own focus change related actions.

### CODE REVIEW

File: Source/x11/XGServerEvent.m [hunk 1]
```diff
@@ -472,10 +472,10 @@ - (void) processEvent: (XEvent *) event
             /*
              * We must hand over control of our icon/miniwindow
              * to Window Maker.
-                 */
+             */
             if ((cWin->win_attrs.window_style
-              & (NSMiniWindowMask | NSIconWindowMask)) != 0
-              && eventType == NSLeftMouseDown && clickCount == 1)
+                 & (NSMiniWindowMask | NSIconWindowMask)) != 0
```
Comment (fredkiefer) [inline Source/x11/XGServerEvent.m]: Something seems to be wrong with the indentation of these two lines. Could you please check?

File: Source/x11/XGServerEvent.m [hunk 2]
```diff
@@ -484,7 +484,8 @@ - (void) processEvent: (XEvent *) event
                 XSendEvent(dpy, cWin->parent, True,
                            ButtonPressMask, &xEvent);
                 XFlush(dpy);
-                break;
+                if (clickCount != 2)
```
Comment (fredkiefer) [inline Source/x11/XGServerEvent.m]: Why don't you check for "clickCount == 1", which would be closer to the old behaviour? This isn#T important but changes the result of a triple click.

File: Source/x11/XGServerEvent.m (hunk 1 above)
Comment (trunkmaster) [reply Source/x11/XGServerEvent.m]: The indentation is correct. The line 477 aligned to the inner expression (`c` character of `cWin`). 
Line 478 aligned to the brackets of the if () statement. This is how emacs make an indentation of C code.

File: Source/x11/XGServerEvent.m (hunk 2 above)
Comment (trunkmaster) [reply Source/x11/XGServerEvent.m]: There was some glitches with old -back and -gui for double-clicks. I've just checked it with current code - everything seems correct.

**Pull request merged**